test/wrb_gas.js

```diff
@@ -665,10 +669,14 @@ contract("WitnetRequestBoard", ([
         "claimer balance should have increased after reporting dr request result by 0.5 eth",
       ).to.equal(true)
       expect(
-        ownerFinalBalance.eq(ownerInitialBalance
+        requestorFinalBalance.eq(requestorInitialBalance
           .add(ether("0.125"))
         ),
-        "Owner balance should have increased after reporting b lock by 0.125 eth",
+        "Requestor balance should have increased after reporting block by 0.125 eth",
```

Comment:
```suggestion
        // Since the block was already reported, the requestor should receive back the block reward
        "Requestor balance should have increased after reporting block by 0.125 eth",
```